### PR TITLE
WAS-585: Add partner page

### PR DESCRIPTION
**Pull request by rickclare**

### CODE REVIEW

**Comment by rickclare** [comment]
Over to @Cruikshanks  for PR review.

File: config/environments/production.rb [hunk 1]
```diff
@@ -90,8 +90,8 @@
 
   # These are the minimum settings
   smtp = {
-    address: ENV["EMAIL_HOST"],
-    port: ENV["EMAIL_PORT"]
+    address: (ENV["EMAIL_HOST"] || fail("Environment variable DEFAULT_URL_HOST has not been set")),
```
Comment (Cruikshanks) [inline config/environments/production.rb]: Do the fail messages need to be updated to use the relevant env var e.g. `EMAIL_HOST`?

Comment (rickclare) [inline config/environments/production.rb]: Well spotted!! I will correct this.

File: spec/features/correspondence_contact_spec.rb [hunk 2]
```diff
@@ -107,8 +107,12 @@
     end
 
     scenario "Back link navigation" do
+      pending "awaiting a more robust solution for back-button testing"
+      fail "pending spec"
+
       # When I click on the back link
       page_object.click_back_link
+      # TODO: finish this zcenario
```
Comment (Cruikshanks) [inline spec/features/correspondence_contact_spec.rb]: If I'm picking up on typo's then I demand this one gets left in. You may just want to suggest the TODO should be read in the style of an *Allo, Allo* character!

Comment (rickclare) [inline spec/features/correspondence_contact_spec.rb]: Good moaning!

Comment (Cruikshanks) [inline spec/features/correspondence_contact_spec.rb]: Just waiting on CI to finish and then I'll push the button.

**Pull request merged**